src/content/structured/accessibility/documenting/accessibility-statements-guidance.mdx

```diff
@@ -44,11 +44,11 @@ The [when to create and update](/accessibility/documenting/when-to-create-and-up
 
 ### How to use the template
 
-You may need to make minor changes such as changing between singular and plural or 'app to service' but don't re-order, change titles or remove anything unless this guidance tells you to.
+You may need to make minor changes, such as changing between singular and plural or 'app to service', but don't re-order, change titles or remove anything unless this guidance tells you to.
```

Comment:
there are a lot of commas in this sentence? It may be worth breaking them up with a full stop.

Reply:
I think it reads OK as is and the commas aid the flow by separating the middle clause. I think breaking into two sentences may increase complexity, so I would prefer to leave as the original sentence if that's OK.